CONTRIBUTING.md

```diff
@@ -11,8 +11,9 @@ These are some of the many ways to contribute:
 * :mag: Fixing typos and improving to the documentation
 * :bulb: Writing code for everyone to use
```

Comment:
I think we want to [CRediT](https://casrai.org/credit) some other ways of contributing :)

```suggestion
* :bulb: Writing code for everyone to use
* :people_holding_hands: Community engagement and outreach
```